cpp/dolfinx/la/SparsityPattern.h

```diff
@@ -67,6 +67,19 @@ class SparsityPattern
 
   /// @brief Insert non-zero locations using local (process-wise)
   /// indices.
+  void insert(std::int32_t row, std::int32_t col);
+
+  /// @brief Insert non-zero locations using local (process-wise)
+  /// indices.
+  ///
+  /// This inserts for every row in rows the columns (listed in cols) to the
+  /// sparsity pattern. This allows for easy setting of dense sub-blocks, for
+  /// example sp.insert({0, 1, 2}, {1, 2, 3}) creates a 3x3 block in the
+  /// sparsity pattern with top left corner at (0, 1).
+  ///
+  /// Note: Especially this operation does not perform a pairwise insertion to
+  /// the sparsity pattern, i.e. the above call does not introduce the entries
+  /// at locations (0, 1), (1, 2) and (2, 3)!
```

Comment:
This is a bit confusing to me, because it does create these entries. Maybe you could say that rows and columns could have different sizes, so the 1-to-1 map could not be implied.

Reply:
How about: "..., i.e. the above call creates more then (9 in total) entries and not only entries at the locations (0,1), (1,2) and (2,3)!"

Reply:
Added a combination of both. I think we want to say $A[row[i], col[j]]$ is being set though.